jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = /** @type {import('@jest/types').Config.InitialOptions} */ ({
+  // all of our tests have a browser and a node process, so the default max workers is too many.
+  maxWorkers: Math.ceil(require('os').cpus().length / 2),
```

Comment:
Should this go to our environment/preset? I expect our users to have the same issue.